plugins/versionpress/src/Git/CommitMessage.php

```diff
@@ -57,6 +57,19 @@ public function getSubject()
     }
 
     /**
+     * Returns commit message subject without prefix set in constant VP_COMMIT_MESSAGE_PREFIX
+     * 
+     * @return string
+     */
+    public function getUnprefixedSubject()
```

Comment:
I think stripping the prefix from the commit message is not a responsibility of the `CommitMessage` class. If `GitRepository` adds the prefix, it should also remove it (probably in the `log` method).